src/react-native/react-native-wizard.ts

```diff
@@ -0,0 +1,256 @@
+
+  const rnVersion = getPackageVersion(RN_PACKAGE, packageJson);
+  if (rnVersion) {
+    await confirmContinueIfPackageVersionNotSupported({
+      packageName: RN_HUMAN_NAME,
+      packageVersion: rnVersion,
+      packageId: RN_PACKAGE,
+      acceptableVersions: SUPPORTED_RN_RANGE,
+    });
+  }
+
+  const { selectedProject, authToken } = await getOrAskForProjectData(
+    options,
+    'react-native',
+  );
+
+  await installPackage({
+    packageName: RN_SDK_PACKAGE,
+    alreadyInstalled: hasPackageInstalled(RN_SDK_PACKAGE, packageJson),
+  });
+
+  addSentryInit({ dsn: selectedProject.keys[0].dsn.public });
+
+  if (fs.existsSync('ios')) {
+    await patchXcodeFiles({ authToken });
+  }
+
+  if (fs.existsSync('android')) {
+    await patchAndroidFiles({ authToken });
+  }
+
+  const confirmedFirstException = await confirmFirstSentryException(
+    selectedProject,
+  );
+
+  if (confirmedFirstException) {
+    clack.outro(
+      `${chalk.green('Everything is set up!')}
+
+   ${chalk.dim(
+     'If you encounter any issues, let us know here: https://github.com/getsentry/sentry-react-native/issues',
+   )}`,
+    );
+  } else {
+    clack.outro(
+      `${chalk.dim(
+        'Let us know here: https://github.com/getsentry/sentry-react-native/issues',
+      )}`,
+    );
+  }
+}
+
+function addSentryInit({ dsn }: { dsn: string }) {
+  const prefixGlob = '{.,./src}';
+  const suffixGlob = '@(j|t|cj|mj)s?(x)';
+  const universalGlob = `App.${suffixGlob}`;
+  const jsFileGlob = `${prefixGlob}/+(${universalGlob})`;
+  const jsPath = getFirstMatchedPath(jsFileGlob);
+  if (!jsPath) {
+    clack.log.warn(
+      `Could not find main App file using ${chalk.bold(jsFileGlob)}.`,
+    );
+    return;
+  }
+  const jsRelativePath = path.relative(process.cwd(), jsPath);
+
+  const js = fs.readFileSync(jsPath, 'utf-8');
+  if (
+    doesJsCodeIncludeSdkSentryImport(js, { sdkPackageName: RN_SDK_PACKAGE })
+  ) {
+    clack.log.warn(`${chalk.bold(jsRelativePath)} already includes Sentry.`);
+    return;
+  }
+
+  const newContent = addSentryInitWithSdkImport(js, { dsn });
+
+  clack.log.success(
+    chalk.greenBright(
+      `Added ${chalk.bold('Sentry.init')} to ${chalk.bold(jsRelativePath)}.`,
+    ),
+  );
+
+  fs.writeFileSync(jsPath, newContent, 'utf-8');
+  clack.log.success(
+    chalk.greenBright(`${chalk.bold(jsRelativePath)} changes saved.`),
+  );
+}
+
+async function confirmFirstSentryException(project: SentryProjectData) {
+  const projectsIssuesUrl = `${project.organization.links.organizationUrl}/issues/?project=${project.id}`;
+
+  clack.note(`To make sure everything is set up correctly, put the following code snippet into your application.
+The snippet will create a button that, when tapped, sends a test event to Sentry.
+
+After that check your project issues:
+
+${chalk.cyan(projectsIssuesUrl)}
+
+<Button title='Try!' onPress={ () => { Sentry.captureException(new Error('First error')) }}/>`);
```

Comment:
l: To make copy/pasting easier, wdyt about using the `showCopyPasteInstructions` and `makeCodeSnippet` helpers? Doesn't strictly have to happen though, so feel free to disregard.

Reply:
I've missed that but it looks better than my current solution. I'll use it.